### DISCUSSION TITLE
Tidepool test servers

### DISCUSSION THREAD

**Discussion by psonnera** [NightscoutFoundation/xDrip#2273] [General]
Is this really necessary anymore? Can it be put in engineering mode?
People will select it by mistake and won't get data in Tidepool...
![image](https://user-images.githubusercontent.com/33809361/184603546-6c12c75a-5892-4d75-aec5-cea354a02a65.png)

**Comment by Navid200** [comment]
I'd rather not add items to engineering mode just because we think they are not needed anymore.  The engineering mode already has too many items in it.
I'm hoping that engineering mode would be cleaned up and I am happy to do the work if approved.
So, if you know this is not needed, I'd rather just remove it altogether.

Is there any scenario where this could still be useful?

**Reply by psonnera** [reply]
@Navid200 I don't even know why this was added... probably during development phase?
